src/CashuWallet.ts

```diff
@@ -85,17 +92,19 @@ class CashuWallet {
 	 * @param invoice
 	 * @param proofsToSend the exact amount to send including fees
 	 * @param feeReserve? optionally set LN routing fee reserve. If not set, fee reserve will get fetched at mint
+	 * @param count? optionally set count to derive secret deterministically. CashuWallet class must be initialized with seed phrase to take effect
```

Comment:
To me, "count" sounds like more like "how many secrets would you like to derive". 

"counter" might be more appropriate for "I want to derive the n-th secret"

Reply:
After reading to the end of the PR, I've noticed that "count" indeed correctly refers to the first option, so I'll take this back! :)

Reply:
It seems like, I must take my taking back bak again.

The term "count" is indeed used in two different ways which leads to the confusion I've mentioned above:

In `restore` it is used in the first meaning (a range).
In `createBlindedMessages` it is used in the second meaning (a number).

Reply:
Plus 1, I wanted to make this same suggestion... Either index or counter. Either convey the correct sentiment.

Reply:
agree! restore should be count, everything else should be counter.